cmd/repo-query/repo-query.go

```diff
@@ -85,7 +76,7 @@ func newRepoQueryAgent(apiClient *api.Client) *repoQueryAgent {
 	}
 }
 
-func (rq *repoQueryAgent) getOwnerAndRepo(url string) (owner, repo string, err error) {
+func GetOwnerAndRepoFromURL(url string) (owner, repo string, err error) {
```

Comment:
Looks like it's getting used in `contributors.go` which is why it was changed from a method on the `repoQueryAgent` struct to a publicly facing package function.

This is a pretty general utility function, so we should abstract it out into some a utility package in `pkg/`. Maybe something like `pkg/github` since it's pretty specific to GitHub